src/app/app.routes.ts

```diff
@@ -1,3 +1,6 @@
 import { Routes } from '@angular/router';
+import { CategoryComponent } from './modules/product/component/category/category.component';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+    {path: "", component: CategoryComponent}
```

Comment:
Esto debe corregirse para poder continuar:
```suggestion
    {path: "categoria", component: CategoryComponent} // también puede se "category". Según como quieras mantener tus rutas: en español o en inglés.
```
De está manera podemos acceder a http://localhost:4200/categoria